packages/ui/browser/src/ts/HtmlComponent.ts

```diff
@@ -2,11 +2,14 @@ import {Destroyable} from './Destroyable';
 import {randomString} from '@0cfg/utils-common/lib/randomString';
 import {has} from '@0cfg/utils-common/lib/has';
 import {RenderLocation} from './RenderLocation';
-import {copy} from '@0cfg/utils-common/lib/copy';
+import {injectable} from 'inversify';
+import {errStatus} from '@0cfg/reply-common/lib/Reply';
+
+const timeoutBeforeWarning: number = 5000;
```

Comment:
nit: It's a good practice for readability to have constants outside of a class in all uppercase.
```suggestion
const TIMEOUT_BEFORE_WARNING: number = 5000;
```